### PR TITLE
Create Books.model.js

### LINKED ISSUE #2: Books

### PR DESCRIPTION
**Pull request by narasimha-1511**
Add the Books Model ..

### CODE REVIEW

**Comment by narasimha-1511** [comment]
fixes #2

**Comment by probablyArth** [comment]
closes #2

File: server/models/Books.model.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Schema, model } from "mongoose";
+
+const UserSchema = new Schema({
+  book_id: { type: mongoose.Schema.Types.ObjectId , required:true },
```
Comment (probablyArth) [inline server/models/Books.model.js]: book_id should not be an ObjectId, it should be string only

**Review by probablyArth — CHANGES_REQUESTED** [review]

**Review by probablyArth — APPROVED** [review]
LGTM!

**Pull request merged**